app/assets/stylesheets/layouts/_footer.scss

```diff
@@ -190,33 +190,3 @@
   max-width: 288px;
   height: 45px;
 }
-
-.l-footer-secondary__list--left {
-  @include column(12);
-  margin-top: baseline-unit(3);
-
-  @include respond-to($mq-l) {
-    @include column(4);
-    margin-top: 0;
-  }
-}
-
-.l-footer-secondary__list--right {
-  @include column(12);
-  padding-bottom: baseline-unit(3);
-
-  @include respond-to($mq-two-col) {
-    display: block;
-  }
-
-  @include respond-to($mq-l) {
-    @include column(8);
-    text-align: right;
-    float: right;
-    padding-bottom: 0;
-  }
-}
-
-.l-footer-secondary {
```

Comment:
This was actually an unnecessary style as it was being called and overridden 3 times. The colour comes through from yeast.. which now has the change to make the colour blue.